Resources/Prototypes/Technologies/sheet.yml

```diff
@@ -1,26 +1,78 @@
+# In order to make this list somewhat organized, please place new technologies underneath their overarching "base" technology.
```

Comment:
You should probably rename this file. "sheet" was a bit of a placeholder name.
Also, maybe you could some technologies in separate files to keep it organized.